src/visualization/src/visualization.C

```diff
@@ -119,6 +119,22 @@ namespace GRINS
     return;
   }
 
+  void Visualization::output_amr
+  ( std::shared_ptr<libMesh::EquationSystems> equation_system,
+    const unsigned int amr_step )
+  {
+    std::stringstream suffix;
+
+    suffix << "amr" << amr_step;
+
+    std::string filename = this->_vis_output_file_prefix;
+    filename+="."+suffix.str();
+
+    this->dump_visualization( equation_system, filename, 0.0 );
+
+    return;
```

Comment:
This was bad style that I'd introduced long ago, but haven't removed from everywhere, but let's not introduce more. For void functions, we just let the closing brace be the end and don't explicitly call `return`.

Reply:
Sorry, too much copy-pasta...